### PR TITLE
Remove Ubuntu 14.04

### PR DESCRIPTION
**Pull request by JtheBAB**
Ubuntu 14.04 is April 2019 End of life ( 
https://www.ubuntu.com/about/release-cycle )

This pull request removes everything related to 14.04

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks for the contribution! Have queued this and a similar PR for RHEL (https://github.com/mattermost/docs/pull/2507) for team discussion on how to best monitor supported OS versions.

**Comment by jasonblais** [comment]
@JtheBAB For these doc changes, wondering your thoughts on adding a note at the top of the document to mention the upcoming end-of-life for Ubuntu 14.04 on April/2019, and then remove the doc once the date has passed?

**Comment by JtheBAB** [comment]
IMHO i think it is fine to remove it now. 

New customers are hopefully installing Mattermost on more recent operating systems and normally IT is not pleased to migrate after 2 Months.

File: source/install/install-ubuntu-1404-mattermost.rst [hunk 1]
```diff
@@ -1,91 +0,0 @@
-.. _install-ubuntu-1404-mattermost:
```
Comment (jasonblais) [inline source/install/install-ubuntu-1404-mattermost.rst]:
```
.. _install-ubuntu-1404-mattermost:

Ubuntu 14.04 LTS reached end of life April, 2019. Please refer to install guides for :doc:`Ubuntu 16.04 LTS <install-ubuntu-1604>` or :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` instead.
```

File: source/install/install-ubuntu-1404-mysql.rst [hunk 2]
```diff
@@ -1,47 +0,0 @@
-.. _install-ubuntu-1404-mysql:
-
-Installing MySQL Database Server
```
Comment (jasonblais) [inline source/install/install-ubuntu-1404-mysql.rst]:
```
.. _install-ubuntu-1404-mysql:

Ubuntu 14.04 LTS reached end of life April, 2019. Please refer to install guides for :doc:`Ubuntu 16.04 LTS <install-ubuntu-1604>` or :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` instead.
```

File: source/install/install-ubuntu-1404-postgresql.rst [hunk 3]
```diff
@@ -1,104 +0,0 @@
-.. _install-ubuntu-1404-postgresql:
-
-Installing PostgreSQL Database Server
```
Comment (jasonblais) [inline source/install/install-ubuntu-1404-postgresql.rst]: Propose leaving this on the page:

```
.. _install-ubuntu-1404-postgresql:

Ubuntu 14.04 LTS reached end of life April, 2019. Please refer to install guides for :doc:`Ubuntu 16.04 LTS <install-ubuntu-1604>` or :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` instead.
```

File: source/install/install-ubuntu-1404-server.rst [hunk 4]
```diff
@@ -1,18 +0,0 @@
-.. _install-ubuntu-1404-server:
-
-Installing Ubuntu Server 14.04 LTS
```
Comment (jasonblais) [inline source/install/install-ubuntu-1404-server.rst]: Propose leaving this on the page:

```
.. _install-ubuntu-1404-server:

Ubuntu 14.04 LTS reached end of life April, 2019. Please refer to install guides for :doc:`Ubuntu 16.04 LTS <install-ubuntu-1604>` or :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` instead.
```

File: source/install/install-ubuntu-1404.rst [hunk 5]
```diff
@@ -1,23 +0,0 @@
-.. _install-ubuntu-1404:
```
Comment (jasonblais) [inline source/install/install-ubuntu-1404.rst]: Propose leaving this on the page:

```
.. _install-ubuntu-1404:

Ubuntu 14.04 LTS reached end of life April, 2019. Please refer to install guides for :doc:`Ubuntu 16.04 LTS <install-ubuntu-1604>` or :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` instead.
```

File: source/install/prod-ubuntu-1404.rst [hunk 6]
```diff
@@ -1,6 +0,0 @@
-..  _prod-ubuntu-1404:
-
-Production Install on Ubuntu 14.04 LTS
```
Comment (jasonblais) [inline source/install/prod-ubuntu-1404.rst]: Propose leaving this on the page:

```
.. prod-ubuntu-1404:

Ubuntu 14.04 LTS reached end of life April, 2019. Please refer to install guides for :doc:`Ubuntu 16.04 LTS <install-ubuntu-1604>` or :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` instead.
```

**Comment by JtheBAB** [comment]
@jasonblais 

I changed it according to your request

**Pull request merged**